website/docs/integrations/vscode.mdx

```diff
@@ -22,72 +23,125 @@ description: ConfigCat Visual Studio Code extension. This is a step-by-step guid
 
+<img src="/docs/assets/vscode/ff_setting_view.gif" className="zoomable" alt="Usage of ConfigCat Feature Flags Visual Studio Code Extension" />
+
 ## Install extension
 
-### Visual Studio Code Marketplace
+### Install from Visual Studio Code Marketplace
 
 1. Open [ConfigCat Feature Flags Extension](https://marketplace.visualstudio.com/items?itemName=ConfigCat.configcat-feature-flags) at Visual Studio Marketplace.
 1. Click on the Install button.
-1. Configure extension (see below)
+1. [Configure extension](#configure-extension).
 
-### or within Visual Studio Code
+### Install within Visual Studio Code
 
-1. Search for ConfigCat Feature Flags at the Extensions page.
+1. Search for ConfigCat Feature Flags on the Extensions page.
 1. Click on the Install button.
-1. Configure extension (see below).
+1. [Configure extension](#configure-extension).
 
-### or install from VSIX file
+### Install from VSIX file
 
 1. Open [ConfigCat Feature Flags Extension](https://marketplace.visualstudio.com/items?itemName=ConfigCat.configcat-feature-flags) at Visual Studio Marketplace.
 1. Click on the Download Extension link.
 1. Click on the More Actions icon on the Extensions page in Visual Studio Code and select Install from VSIX...
-1. Configure extension (see below).
+1. [Configure extension](#configure-extension).
 
 ## Configure extension
 
 ### Authentication
 
-<img src="/docs/assets/vscode/auth.gif" className="zoomable" alt="Installation of ConfigCat Feature Flags Visual Studio Code Extension" />
-
 1. Get your ConfigCat Public API credentials from [ConfigCat Dashboard/Public Management API credentials](https://app.configcat.com/my-account/public-api-credentials).
 1. Authenticate ConfigCat in Visual Studio Code by
    - clicking on the ConfigCat Feature Flags icon on the Activity Bar and clicking on any of the Authenticate buttons.
    - or running the `ConfigCat - Log In` command from the Command Palette.
 
+ <img src="/docs/assets/vscode/auth.gif" className="zoomable" alt="Installation of ConfigCat Feature Flags Visual Studio Code Extension" />
+
 ### Advanced
 
-If you are trying to use the ConfigCat Feature Flags extension for Visual Studio Code with a `dedicated hosted`/[on-premise](https://configcat.com/on-premise/) ConfigCat instance, you can specify your custom urls by executing the `Preferences: Open Workspace Settings` command from the Command Palette and search for `Extensions/ConfigCat` or clicking the manage button on the ConfigCat Feature Flags extension's page. Important settings:
+If you are trying to use the ConfigCat Feature Flags extension for Visual Studio Code with a `dedicated hosted`/[on-premise](https://configcat.com/on-premise/) ConfigCat instance. In that case, you can specify your custom URLs by executing the `Preferences: Open Workspace Settings` command from the Command Palette and searching for `Extensions/ConfigCat` or clicking the manage button on the ConfigCat Feature Flags extension's page. Important settings:
```

Comment:
Szuper, hogy kettétörted azt a nagy mondatot!

Viszont így az első mondatnak önmagában nincs igazi tartalma.

És én "a you are trying to use" helyett is "you use"-t használnék, mert a nem anyanelyvi angoloknak sokat segít.

Valami ilyesmit javaslok:

This section is for you if you use a dedicated hosted`/[on-premise](https://configcat.com/on-premise/) ConfigCat instance. In that case, you must specify your custom ConfigCat URLs in Visual Studio Code. You can do that by executing the `Preferences: Open Workspace Settings` command from the Command Palette and searching for `Extensions/ConfigCat` or clicking the manage button on the ConfigCat Feature Flags extension's page. Important settings: